fluss-connectors/fluss-connector-flink/src/test/java/com/alibaba/fluss/connector/flink/sink/FlinkTableSinkITCase.java

```diff
@@ -378,6 +380,30 @@ void testInsertWithoutSpecifiedCols() {
                 .isEqualTo(expectPlan);
     }
 
+    @Test
+    void testIgnoreDelete() throws Exception {
+        String cdcSourceData =
+                TestValuesTableFactory.registerData(
+                        Arrays.asList(
+                                Row.ofKind(RowKind.INSERT, 1, 3501L, "Tim"),
+                                Row.ofKind(RowKind.DELETE, 1, 3501L, "Tim"),
+                                Row.ofKind(RowKind.INSERT, 2, 3502L, "Fabian"),
+                                Row.ofKind(RowKind.UPDATE_BEFORE, 2, 3502L, "Fabian"),
+                                Row.ofKind(RowKind.UPDATE_AFTER, 3, 3503L, "coco")));
+        tEnv.executeSql(
```

Comment:
We also should verify a stream with -D/-U change mode can write to a log sink with `ignoreDelete`, but currently the source's changelog mode is +I only...

Btw, can we just use the public interface `tableEnv.fromChangelogStream` to do the testing? I'd like not to introduce an external test jar `flink-table-planner` since the test jar may change overtime and we may need to adjust our test then..